cylc/flow/network/server.py

```diff
@@ -148,89 +149,122 @@ def __init__(self, schd, context=None, barrier=None,
             IgnoreFieldMiddleware,
         ]
 
-    def _socket_options(self):
-        """Set socket options.
+        self.queue = Queue()
+        self.publish_queue = Queue()
+        self.stopping = False
+        self.stopped = True
 
-        Overwrites Base method.
+    def configure(self):
```

Comment:
Is there any reason to keep this as a separate method and not put all of this straight into `__init__()`?

Reply:
Yes no reason, configure was called right after server instantiation... Combined.

Reply:
👍  But now there are several properties that are instantiated as `None` only to be redefined a few lines down, like `self.zmq_context`. Would be nice to tidy those up.